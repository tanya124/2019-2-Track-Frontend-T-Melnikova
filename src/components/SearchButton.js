import React from 'react';
import styled from '@emotion/styled';
import search from '../assets/search.svg';

const Button = styled.img`
	width: 3em;
	margin-right: 10px;
	&:hover {
		filter: invert(1);
	}
`;


function SearchButton() {
	return (
		<Button src={ search } />
	);
}

export default SearchButton;