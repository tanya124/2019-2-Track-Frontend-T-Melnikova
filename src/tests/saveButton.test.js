import React from 'react'
import renderer from 'react-test-renderer'
import SearchButton from '../components/Buttons/SearchButton'

it('renders correctly', () => {
  const tree = renderer.create(<SearchButton />).toJSON()
  expect(tree).toMatchSnapshot()
})
