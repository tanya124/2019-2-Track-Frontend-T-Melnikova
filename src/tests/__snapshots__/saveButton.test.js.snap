// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<img
  className="css-6jpwhn"
  src="search.svg"
/>
`;
